<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFileUtilities">
    <property name="title">
     <string>File utilities</string>
    </property>
    <addaction name="actionLockFileUtility"/>
    <addaction name="actionADSEditor"/>
    <addaction name="actionCreateMutex"/>
   </widget>
   <widget class="QMenu" name="menuDomainUtilities">
    <property name="title">
     <string>Domain utilities</string>
    </property>
    <addaction name="actionCheckDomainUtility"/>
   </widget>
   <addaction name="menuFileUtilities"/>
   <addaction name="menuDomainUtilities"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLockFileUtility">
   <property name="text">
    <string>Lock file</string>
   </property>
  </action>
  <action name="actionADSEditor">
   <property name="text">
    <string>ADS editor</string>
   </property>
  </action>
  <action name="actionDomain_utilities">
   <property name="text">
    <string>Domain utilities</string>
   </property>
  </action>
  <action name="actionCheckDomainUtility">
   <property name="text">
    <string>Check domain utility</string>
   </property>
  </action>
  <action name="actionCreateMutex">
   <property name="text">
    <string>Create mutex</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
